src/worker/api/index.ts

```diff
@@ -1607,10 +1608,28 @@ const getTradingBlockOffers = async ({
 			"noCopyCache",
 		);
 		const stats = bySport({
-			basketball: ["gp", "min", "pts", "trb", "ast", "per"],
-			football: ["gp", "keyStats", "av"],
-			hockey: ["gp", "keyStats", "ops", "dps", "ps"],
+			basketball: [
+				"stat:gp",
+				"stat:min",
+				"stat:pts",
+				"stat:trb",
+				"stat:ast",
+				"stat:per",
+			],
+			football: ["stat:gp", "stat:keyStats", "stat:av"],
+			hockey: ["stat:gp", "stat:keyStats", "stat:ops", "stat:dps", "stat:ps"],
 		});
+		const config: TableConfig = new TableConfig("tradingBlock", [
```

Comment:
`const config: TableConfig =` can just be const config =` - in general, only manually specify types when necessary.